redfish-core/lib/account_service.hpp

```diff
@@ -1175,6 +1196,25 @@ inline void getAcfProperties(
     }
     asyncResp->res.jsonValue["Oem"]["IBM"]["ACF"]["ACFInstalled"] =
         acfInstalled;
+
+    crow::connections::systemBus->async_method_call(
+        [asyncResp](const boost::system::error_code ec,
+                    const std::variant<bool>& retval) {
+            if (ec)
+            {
+                BMCWEB_LOG_ERROR << "D-Bus responses error: " << ec;
+                messages::internalError(asyncResp->res);
+                return;
+            }
+            const bool* allowed = std::get_if<bool>(&retval);
```

Comment:
Please check if this is nullptr like 1790

            if (allowUnauthACFUpload == nullptr)
            {
                BMCWEB_LOG_ERROR << "nullptr for allow_unauth_upload";
                messages::internalError(asyncResp->res);
                return;
            }

Reply:
Good catch!  I'll add the check.